kagiso_auth/tests/unit/test_models.py

```diff
@@ -520,3 +521,41 @@ def test_record_sign_out_invalid_status_code_raises(self):
         with pytest.raises(AuthAPIUnexpectedStatusCode):
             did_sign_out = user.record_sign_out()
             assert not did_sign_out
+
+    def test_age_returns_none_when_profile_is_none(self):
+        user = KagisoUser(
+            email='test@email.com',
+        )
+
+        assert user.age is None
+
+    def test_age_returns_none_when_birth_date_doesnt_exist(self):
+        user = KagisoUser(
```

Comment:
You right, I mean't to actually add the profile attribute to the second test, as per my test name suggested, thanks for the spot